protected/views/patient/_form_address.php

```diff
@@ -60,7 +60,8 @@
 <div class="row field-row">
     <div class="large-3 column"><?php echo $form->labelEx($address,'country_id'); ?></div>
     <div class="large-4 column end">
-        <?php echo $form->dropDownList($address,'country_id', $countries, array('empty'=>'-- select --')); ?>
+
+        <?php echo $form->dropDownList($address,'country_id', $countries, array('options'=>array(array_search('Australia', $countries)=>array('selected'=>true)))); ?>
```

Comment:
I'm wondering if there is a simpler way to do this. That said, no need to do so until after today's release.